BE/src/main/java/team07/airbnb/controller/ReviewController.java

```diff
@@ -58,11 +53,7 @@ public List<ReviewWithReplyResponse> getReviews(@PathVariable Long accommodation
     @PostMapping("/{bookingId}")
     @ResponseStatus(OK)
     public void postReview(@PathVariable Long bookingId, @RequestBody ReviewPostRequest request, TokenUserInfo user) {
-        if (!bookingService.isUserBookerOf(bookingId, userService.getCompleteUser(user))) {
-            throw new UnAuthorizedException(ReviewController.class, user.id(), "ID : {%d} 유저가 자신의 예약이 아닌 예약에 리뷸를 남기려고함".formatted(user.id()));
-        }
-
-        bookingService.addReview(bookingId, user.id(), new ReviewEntity(bookingService.findByBookingId(bookingId), request.content(), request.rating()));
+        bookingInquiryService.addReview(bookingId, user.id(), new ReviewEntity(bookingInquiryService.findByBookingId(bookingId), request.content(), request.rating()));
```

Comment:
1) 보기좋은 코드는 읽기도 좋을꺼같아요, 물론 이건 취향차이일꺼같긴 합니다

```suggestion
        bookingInquiryService.addReview(
            bookingId, user.id(), 
            new ReviewEntity(
                bookingInquiryService.findByBookingId(bookingId), 
                request.content(), 
                request.rating()
            )
        );
```

2) 컨트롤러 내부에서 서비스-서비스를 호출하고 있는데요, 요거가 그렇다면 어느정도는 컨트롤러에서 로직을 담당하고 있다는 생각이 듭니다. >> 해당 로직을 서비스레이어로 위임해주면 어떨까요? 

3) 아마 이런생각이 드실껀데요 DTO(~Request객체) 가 서비스 레이어까지 내려가도 되나? 이런것들 말이죠! 그리고 저의 의견은 Request 가 Service 레이어까지 전파되어도 된다고 생각합니다. (정답이 정해져있는건 아닙니다!)

Reply:
**3번!!** 이번에 정확히 그 고민을 했었고 , 가능하면 내려보내지 않도록 했는데요 , 
요청 데이터의 형태에 서비스까지 영향 받지 않는다는 것이 그 이유였습니다.

**요청 DTO의 필드가 추가되거나 제거되는 상황이 온다면?
-> 비즈니스 로직 상 필요한 정보가 추가되거나 제거된 것
-> 서비스의 변경은 필연적**

과 같이 생각하면 `서비스 레이어까지 전파되어도 된다`에 저도 어느정도 동의가 되는 것 같아요